### PR TITLE
Update `TabularPreset` parameters

### LINKED ISSUE #887: TabularPreset doesn't support new sampling parameters
### Environment Details

Please indicate the following details about the environment in which you found the bug:

* SDV version: 0.16.0.dev1

### Error Description

When I create a `TabularPreset` model the sampling method parameters don't behave as expected:

- `max_tries_per_batch` cannot be used on any of the methods
- `batch_size` cannot be used on `sample_conditions` or `sample_remaining_columns` (although it works on the generic `sample` method)

### Additional Notes
- Certain if/else blocks can now be unified since the parameter names are the same for all sampling methods. eg.
https://github.com/sdv-dev/SDV/blob/master/sdv/lite/tabular.py#L238-L247

### PR DESCRIPTION
**Pull request by fealho**
Resolve #887.

### CODE REVIEW

**Review by pvk-developer — APPROVED** [review]
LGTM!

File: tests/integration/lite/test_tabular.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+def test_sample():
+    """Test sampling for the ``TabularPreset``."""
+    # Setup
+    data = pd.DataFrame({'a': [1, 2, 3]})
+
+    # Run
+    preset = TabularPreset(name='FAST_ML')
+    preset.fit(data)
+    samples = preset.sample(10)
```
Comment (amontanez24) [inline tests/integration/lite/test_tabular.py:14]: can we pass some parameters like the batch size and max tries per batch

**Review by amontanez24 — APPROVED** [review]
One small comment then :shipit:

**Review by amontanez24 — APPROVED** [review]
:shipit:

**Pull request merged**